test/libsolidity/semanticTests/viaYul/array.sol

```diff
@@ -5,13 +5,35 @@ contract C {
 		storageArray.length = 7;
 		require(storageArray.length == 7);
 
+		for (uint i = 0; i < 7; i++)
```

Comment:
Please don't make the tests that complex. If it fails somewhere, it is much harder to check where the actual error is.

Reply:
You mean without loops and just using a few constants to check specific indicies?

Reply:
one test file for length access, another test file for index access. Of course you need to change the length to allow for index access, but if length is fine and index access fails, we know that actually setting the length is not the problem.